draft-ietf-masque-h3-datagram.md

```diff
@@ -357,6 +357,17 @@ the same order.
 Endpoints which receive a Capsule with an unknown Capsule Type MUST silently
 drop that Capsule.
 
+## Error Handling
+
+Each capsule's payload MUST contain exactly the fields identified in its
+description. A capsule payload that contains additional bytes after the
+identified fields or a frame payload that terminates before the end of the
+identified fields MUST be treated as a stream error of type H3_FRAME_ERROR. In
```

Comment:
as I said on the issue, this isn't an HTTP/3 frame error. Furthermore, we need to cover error handling for all HTTP versions; in H2 and H3 that's a stream error, in HTTP/1.1 it means tearing down the whole thing (which I think is acceptable because there is no way to recover the data stream) at this point.